.github/workflows/main.yml

```
@@ -102,29 +92,29 @@ jobs:
-      with:
-        root_options: -v
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-        git_committer_name: 'gh-actions-semver'
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.SEMVERPOLICE_ID }}
+          private-key: ${{ secrets.SEMVER_APP_KEY }}
+
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v2
```

actionlint

the runner of "actions/setup-python@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)